Cake.BenchmarkDotNet/Dto/ConfidenceInterval.cs

```diff
@@ -3,11 +3,11 @@
     public class ConfidenceInterval
     {
         public int N { get; set; }
-        public double Mean { get; set; }
-        public double StandardError { get; set; }
+        public long Mean { get; set; }

```

Comment:
Looking at the report you uploaded, these should still be doubles.

Reply:
Fixed